src/Core/View.js

```diff
@@ -636,6 +637,75 @@ class View extends THREE.EventDispatcher {
         return results;
     }
 
+    /**
+     * Returns features under the mouse, for a set of specified layers.
+     *
+     * @param {MouseEvent|Object} mouseOrEvt - A MouseEvent, or a screen
+     * coordinates.
+     * @param {number} [radius=3] - The precision of the picking, in pixels.
+     * @param {Layer[]} [where] - The layers to look into. If not specified, all
+     * `GeometryLayer` and `ColorLayer` layers of this view will be looked in.
+     *
+     * @return {Object} - An object, with a property per layer. For example,
+     * looking for features on layers `wfsBuilding` and `wfsRoads` will give an
+     * object like `{ wfsBuilding: [...], wfsRoads: [] }`. Each property is made
+     * of an array, that can be empty or filled with found features.
+     *
+     * @example
+     * view.pickFeaturesAt({ x, y });
+     * view.pickFeaturesAt({ x, y }, 1, ['wfsBuilding']);
+     * view.pickFeaturesAt({ x, y }, 3, ['wfsBuilding', myLayer]);
+     */
+    pickFeaturesAt(mouseOrEvt, radius = 3, where = []) {
+        const result = {};
+        let layers = where.length == 0 ? this.getLayers(l => (l.isGeometryLayer || l.isColorLayer)) : where;
+        layers = layers.map((l) => {
+            const id = l.isLayer ? l.id : l;
+            result[id] = [];
+            return id;
+        });
+
+        let precision;
+        const precisions = {
+            PX: radius,
+            M: this.isGlobeView && this.controls ? this.controls.pixelsToMeters(radius) : radius,
+            D: this.isGlobeView && this.controls ? this.controls.pixelsToDegrees(radius) : 0,
+        };
+
+        // Get the mouse coordinates to the correct system
+        const positionVector = new THREE.Vector3();
```

Comment:
could you instance the `positionVector` and `position` before function?

Reply:
Yes !